pages/release-notes/20181115-1.13.md

```diff
@@ -38,20 +38,17 @@ The keepalived charm can be used to run multiple kube-api-loadbalancers behind a
  - Added support for ELB service-linked role ([Issue](https://github.com/juju-solutions/charm-aws-integrator/pull/29))
  - Fixed EasyRSA does not run as an LXD container on 18.04 ([Issue](https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/654))
  - Fixed ceph volumes cannot be attached to the pods after 1.12 ([Issue](https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/662))
-- Fixed ceph volumes fail to attach with "node has no NodeID annotation" ([Issue](https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/675))
-- Fixed ceph-xfs volumes failing to format due to "executable file not found in $PATH" ([Issue](https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/668))
-- Fixed ceph volumes not detaching properly ([Issue](https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/669))
-- Fixed ceph-csi addons not getting cleaned up properly ([Issue](https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/680))
- - Fixed Kubernetes dashboard not showing when running kubectl cluster-info ([Issue](https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/661))
```

Comment:
I removed the `Fixed Kubernetes dashboard not showing when running kubectl cluster-info` item because that was closed with no fix.